src/lib/y2storage/devicegraph.rb

```diff
@@ -303,6 +303,13 @@ def filesystems
       Filesystems::Base.all(self)
     end
 
+    # All mount points in the devicegraph, in no particular order
+    #
+    # @return [Array<MountPoint>]
+    def mount_points
```

Comment:
NP: This method does not have a unit test.

Reply:
Added. More than 40 lines of testcase for literally a one-liner.